### ISSUE TITLE
SSIM metric

### ISSUE THREAD

**Issue by lazrak-mouad** [hukkelas/DeepPrivacy#24]
Hello, Thank you for this awesome work.

So, I'm trying to compute the SSIM metric between real and generated images inside the loss.py program to see it's evolution while training the model.

I tried to do this by transforming real_data tensors to RGB image but the result was not satisfying. So I was wondering if there's a way to get the images (in a pixelwise structure), since SSIM is using pixels not tensors.

Finally, kindly share any ideas or tips to complete this task.

**Comment by lazrak-mouad** [comment]
Udate :

I've just found that I can compute SSIM from tensors using the "pytorch-msssim" module, but I still want to visualise those images.

For now I can display real and generated images, but as I said the results are not satisfying. Here you can see my achievement.
![image_f](https://user-images.githubusercontent.com/40429075/89992217-43887980-dc85-11ea-8194-47eaf4dd4eb8.png)
![image_r](https://user-images.githubusercontent.com/40429075/89992220-44b9a680-dc85-11ea-9eb7-9dbe93d1efb6.png)

**Comment by hukkelas** [comment]
Hi,

Looks like yout issue got solved! 
For the generated imag quality, the network requires a large amount of training to achieve satisfactory results (see paper).
